src/v/raft/heartbeat_manager.cc

```diff
@@ -217,6 +232,8 @@ void heartbeat_manager::process_reply(
             continue;
         }
         auto meta = groups.find(m.group)->second;
+        (*it)->suppress_heartbeats(
```

Comment:
ok more questions. 

This method says: `supress_heartbeats( actually_do_not_supress_them )` ... so which way is it? is the bool class name correct or how should I reason about this method?